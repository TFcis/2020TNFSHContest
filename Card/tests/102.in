10
1 2 3 4 5 6 7 8 9 10
9 5 7 6 4 3 1 2 8 10
1
