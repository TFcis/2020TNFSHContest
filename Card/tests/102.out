7 8 6 5 2 4 3 9 1 10
